README.md

```diff
@@ -262,35 +262,36 @@ spec:
           description: >-
             How often do you want to check the service health?
           format: duration
           default: 30
-          required: false
       - type: question
         name: var2
+        required: true
         input:
           type: string
           title: Alert Pipeline
           description: >-
             How do you want to be alerted for failures detected by this pipeline (e.g. Slack or Microsoft Teams)?
           ref: core/v2/pipeline/metadata/name
           filter: .metadata.labels.provider == "alerts"
-          required: true
     ```
 
     The following `input` fields may be configured:
 
     * **`type`** (required): data type; allowed values: `string`, `int`, `bool`.
-    * **`title`** (required): input field title/label, displayed above the input field.
-    * **`description`** (optional): input field description, displayed below the input field.
+    * **`name`** (required): variable name to be used in `resource_patches` templates.
     * **`required`** (required): indicates whether a user-input is required.
-    * **`format`** (optional): input value display format; allowed values: `sh`, `ecmascript-5.1`, `cron`, `duration`, `tel`, `email`, `url`, `hostname`, `ipv4`, `ipv6`, `envvar`, `sha-256`, `sha-512`, `io.sensu.selector`. Some display formats provide helpers to simplify user input.
-    * **`ref`** (optional): Sensu API resource reference in `<api_group>/<api_resource>/<api_field_path>` format. For example, `core/v2/Pipeline/metadata/name` refers to `core/v2` API group `Pipeline` resources, which will be presented to the user in a drop-down selector; once selected, the value of the `metadata/name` field will be captured as the input value.
-    * **`filter`** (optional): Sensu API resource reference filters in [Sensu Query Expression (SQE)] format; e.g. `.labels.provider == "alerts"`. Used to filter the results of a `ref`.
+    * **`input.title`** (required): input field title/label, displayed above the input field.
+    * **`input.description`** (optional): input field description, displayed below the input field.
+    * **`input.format`** (optional): input value display format; allowed values: `sh`, `ecmascript-5.1`, `cron`, `duration`, `tel`, `email`, `url`, `hostname`, `ipv4`, `ipv6`, `envvar`, `sha-256`, `sha-512`, `io.sensu.selector`. Some display formats provide helpers to simplify user input.
+    * **`input.ref`** (optional): Sensu API resource reference in `<api_group>/<api_resource>/<api_field_path>` format. For example, `core/v2/Pipeline/metadata/name` refers to `core/v2` API group `Pipeline` resources, which will be presented to the user in a drop-down selector; once selected, the value of the `metadata/name` field will be captured as the input value.
+    * **`input.filter`** (coming soon): Sensu API resource reference filters in [Sensu Query Expression (SQE)] format; e.g. `.labels.provider == "alerts"`. Used to filter the results of a `ref`.
```

Comment:
Let's go with `refFilter`

Reply:
@jamesdphillips updated the readme and a few integrations that have this property set in their prompts; can you look at that commit and let me know if the refFilter values look right (so you'll have something to test)?

Reply:
appears to be correct!

Reply:
awesome; thx! 🤞